db/360/SceSblPostSsMgr.yml

```diff
@@ -4,6 +4,13 @@ modules:
   SceSblPostSsMgr:
     nid: 0xB6C941F2
     libraries:
+      SceSblFwLoaderForDriver:
+        kernel: true
+        nid: 0x6FE424E4
+        functions:
+          ksceSblFwLoaderLoad: 0x91C73A54
+          sceSblFwLoaderLock: 0xA6278D27
+          sceSblFwLoaderUnlock: 0xBB59FC7A
```

Comment:
last thing.
imo, these two functions also have to be started with `ksce` prefix.